Ok, this ain't fixed:

```
Traceback (most recent call last):
  File "/work/make-checkout-workdir/test/verify/check-connection", line 1233, in testReverseProxy
    b.open(f"https://{b.address}:{b.port}/system")
    ~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work/make-checkout-workdir/test/common/testlib.py", line 378, in open
    self.bidi("browsingContext.navigate", context=self.driver.context, url=href, wait="complete", timeout=30)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work/make-checkout-workdir/test/common/testlib.py", line 337, in bidi
    raise Error(str(e)) from None
testlib.Error: unknown error: Error: NS_ERROR_NET_INTERRUPT
```

--- a/test/verify/check-connection
+++ b/test/verify/check-connection
@@ -1228,7 +1228,13 @@ until pgrep -f '^(/usr/[^ ]+/[^ /]*python[^ /]* )?/usr/bin/cockpit-bridge'; do s
         m.spawn(f"runuser -u cockpit-session-socket -- {self.ws_executable} --for-tls-proxy -p 9099 -a 127.0.0.1",
                 "ws-fortlsproxy.log")
         m.wait_for_cockpit_running(tls=True)
-        b.open(f"https://{b.address}:{b.port}/system")
+        # Firefox does not handle connection interruptions well, retry once when opening fails.
+        try:
+            b.open(f"https://{b.address}:{b.port}/system")
+        except testlib.Error as exc:
+            if 'NS_ERROR' in str(exc):
+                b.open(f"https://{b.address}:{b.port}/system")
+
         b.wait_visible("#login")
         b.set_val("#login-user-input", "admin")
         b.set_val("#login-password-input", "foobar")

